classes/class.ilAdobeConnectXMLAPI.php

```diff
@@ -1492,7 +1494,9 @@ public function updateMeetingParticipant($meeting_id, $login, $session, $permiss
 			'permission-id'=> $permission
 		));
 
-		$result = file_get_contents($url);
+    		$ctx = $this->proxy(array());
+		$xml_string = file_get_contents($url, false, $ctx);
+		$result = file_get_contents($url, false, $ctx);
```

Comment:
Why do you request the URL twice?

Reply:
**shame on me**
that is a copy-&-paste mistake..